test_runner/fixtures/zenith_fixtures.py

```diff
@@ -2066,6 +2071,10 @@ def check_restored_datadir_content(test_output_dir: str, env: ZenithEnv, pg: Pos
 
 
 def wait_for(number_of_iterations: int, interval: int, func):
+    """
+    Wait until 'func' returns successfully, without exception. Returns the last return value
```

Comment:
probably worth renaming the hole thing to wait_until

Reply:
Done!